metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
This isn't really a where it is loaded problem.  It is a production deployment with no assumptions vs a default virtual box deployment or a 'demo' environment.  And the issue is not new to this PR.  The pre-pr behavior installs bro yaf and snort, and most likely has non-default things in there.  If it is OK with you, I think the Demo System Feature should be another jira.   The decision then is do we keep the status quo ante behavior wrt the default bro, yaf and snort or not?  I will remove the archetype enrichments.... or maybe add a sample configurations resource... I want to think about it

I think we should add a specific 'demo' environment that installs the non-default configurations, and etc.  Explicitly the demo env as a feature.